compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/BIRPackageSymbolEnter.java

```diff
@@ -405,15 +405,15 @@ private void defineFunction(DataInputStream dataInStream) throws IOException {
         BInvokableType funcType = (BInvokableType) readBType(dataInStream);
         BInvokableSymbol invokableSymbol =
                 Symbols.createFunctionSymbol(flags, names.fromString(funcName), names.fromString(funcOrigName),
-                                             this.env.pkgSymbol.pkgID, funcType, this.env.pkgSymbol,
-                                             Symbols.isFlagOn(flags, Flags.NATIVE), pos, toOrigin(origin));
+                        this.env.pkgSymbol.pkgID, funcType, this.env.pkgSymbol,
```

Comment:
Don't re-format existing code that you are not updating

Reply:
Reverted in [#0763](https://github.com/ballerina-platform/ballerina-lang/pull/40901/commits/076383542895d77de1ad438372cf151195500550)